backend/typescript/services/implementations/emailService.ts

```diff
@@ -125,6 +147,7 @@ class EmailService implements IEmailService {
         uniqueCampers.length
       } campers x ${totalCampDays} total days of camp) </li> 
         <li><b>Early drop-off and late pick-up fees:</b> $${dropoffAndPickupFees} </li>
+        <li><b>Discount Amount:</b> $${discountAmount} </li> 
```

Comment:
`{if discountAmount > 0 <li><b>Discount Amount:</b> $${discountAmount} </li> : null}`

(we don't want to show the discount line if the client didn't apply a discount)

Reply:
Thanks for the catch! It's been fixed in the latest commit.